### ISSUE TITLE
no-multi-spaces rule do not get auto-fixed

### ISSUE THREAD

**Issue by nelson6e65** [buzinas/tslint-eslint-rules#381]
```json
    "no-multi-spaces": [
      true,
      {
        "exceptions": {
          "PropertyAssignment": false
        }
      }
    ]
```

Is detecting the error, but is not auto-fixed. 

```js
import { Observable ,  Subject } from 'rxjs';
```

**Comment by Rejpal** [comment]
have exactly same problem
